src/woo_publications/publications/tests/factories.py

```diff
@@ -12,6 +14,8 @@
 class PublicationFactory(factory.django.DjangoModelFactory[Publication]):
     publisher = factory.SubFactory(OrganisationFactory, is_actief=True)
     officiele_titel = factory.Faker("word")
+    bron_bewaartermijn = factory.Faker("word")
+    archiefactiedatum = date.today() + relativedelta(years=10)
```

Comment:
```suggestion
```
the field is nullable, so let's stick with `None`/unspecified to ensure our code can handle this.